tests/qunit/index.html

```diff
@@ -1369,7 +1378,7 @@ <h2 class="upload-instructions drop-instructions">Drop files anywhere to upload<
 			<div class="favicon">
 				<img id="preview-favicon" src="{{ data.url }}" alt="Preview as a browser icon"/>
 			</div>
-			<span class="browser-title" aria-hidden="true">WordPress Develop</span>
+			<span class="browser-title" aria-hidden="true">WordPress Develops22s</span>
```

Comment:
I noticed that too, but when I rebuilt the test file it seemed to clear up for me.